website/server/controllers/api-v3/tasks.js

```diff
@@ -16,6 +16,27 @@ import logger from '../../libs/api-v3/logger';
 
 let api = {};
 
+async function _checkShortNameUniqueness (user, tasks) {
+  let shortNames = tasks.map(task => task.shortName).filter(name => name);
+
+  if (shortNames.length !== [...new Set(shortNames)].length) {
```

Comment:
it took me a moment to understand what the `Set` was used for :) A comment here would help